README.md

```diff
@@ -81,3 +115,4 @@ Still is released under the [ISC License](./LICENSE).
 [hex-badge]: https://img.shields.io/hexpm/v/still?style=flat-square
 [build-badge]: https://img.shields.io/github/workflow/status/still-ex/still/Elixir%20CI?style=flat-square
 [docs-badge]: https://img.shields.io/badge/-docs-informational?style=flat-square
+[issue-tracker]: https://github.com/still-ex/still/issues
```

Comment:
@gabrielpoca are we keeping Trello up to date? Does it make sense to list it instead?

Reply:
@frm we are not, and to be honest I would rather keep everything in github. It's a bit disorganized, but I think it's working better than having another tool. What do you think?